bin/bcs-mumble-auth.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cd /opt/bcs/bcs-mumble/src
+. ../bin/activate
+
+echo "__import__('brave.mumble.service').mumble.service.main()" | paster shell local.ini
```

Comment:
We also need to include the config things we do in brave production, IE paster config files for flup sockets and storing the config in a separate config folder. Will do that work once this is pulled.